src/resources/ticket/controller.js

```diff
@@ -16,24 +16,40 @@ const getAllTickets = async (req, res) => {
 
 
 const getCompetitionFromOneTicket = async (req, res) => {
- 
+
   try {
-  const data = await prisma.ticket.findUnique({   //You can use .findFirst and .findUnique
-  where: {
-    id: parseInt(req.params.id)
-  },  
-     
-            include: {
-              competition: true
-            }
-     
-  })
-  res.json(data)
-} catch (error) {
-  console.error("[ERROR] getAll: ", { error });
-  res.json({ error });
-}
+    const data = await prisma.ticket.findUnique({   //You can use .findFirst and .findUnique
+      where: {
+        id: parseInt(req.params.id)
+      },
+
+      include: {
+        competition: true
+      }
+
+    })
+    res.json(data)
+  } catch (error) {
+    console.error("[ERROR] getAll: ", { error });
+    res.json({ error });
+  }
 }
 
+function createTicket(req, res) {
+  console.log(req.body)
+  prisma.ticket.create({
+    data: {
+      ...req.body,
+    },
+  })
+    .then((result) => {
+      console.log(result)
+      res.json({ data: result })
```

Comment:
This is breaking consistency a little. If you check the function above, "getCompetitionFromOneTicket", it's doing `res.json(data)` - that's passing the data returned from prisma directly into the json. What you're doing here is `res.json({ data: result })`, which is passing in an object with a `data` property that holds the data returned from prisma.

It's not a huge deal, but it'll help the whole team if you keep to the same format

Reply:
shall I edit and re-push?